examples/rust/external_data_loader/src/main.rs

```diff
@@ -0,0 +1,90 @@
+//! Example of an external data-loader executable plugin for the Rerun Viewer.
+
+use rerun::MediaType;
+
+const USAGE: &str = "
+This is an example executable data-loader plugin for the Rerun Viewer.
+
+It will log Rust source code files as markdown documents.
+To try it out, install it in your $PATH (`cargo install --path . -f`), then open a Rust source file with Rerun (`rerun file.rs`).
+
+USAGE:
+  rerun-loader-rust-file [OPTIONS] FILEPATH
+
+FLAGS:
+  -h, --help                    Prints help information
+
+OPTIONS:
+  --recording-id RECORDING_ID   ID of the shared recording
+
+ARGS:
+  <FILEPATH>
+";
+
+#[allow(clippy::exit)]
+fn usage() -> ! {
+    eprintln!("{USAGE}");
+    std::process::exit(1);
+}
+
+// The Rerun Viewer will always pass these two pieces of information:
+// 1. The path to be loaded, as a positional arg.
+// 2. A shared recording ID, via the `--recording-id` flag.
+//
+// It is up to you whether you make use of that shared recording ID or not.
+// If you use it, the data will end up in the same recording as all other plugins interested in
+// that file, otherwise you can just create a dedicated recording for it. Or both.
+struct Args {
+    filepath: std::path::PathBuf,
+    recording_id: Option<String>,
+}
+
+impl Args {
+    fn from_env() -> Result<Self, pico_args::Error> {
```

Comment:
@jprochazk just started using `argh` for our build scripts (compiles much faster than `clap`); you might want to use that here too for consistency (and so we don't have three different argument parsers)